engine/src/main/resources/assets/shaders/hdr_frag.glsl

```diff
@@ -31,11 +31,7 @@ void main(){
 #endif
 
 #ifdef UNCHARTED_2_TONEMAP
-    color.rgb *= exposure;
-    vec3 adjColor = uncharted2Tonemap(color.rgb);
-    vec3 whiteScale = 1.0/uncharted2Tonemap(vec3(whitePoint));
-    vec3 finalColor = adjColor*whiteScale;
-    color.rgb = finalColor;
+    color.rgb = uncharted2Tonemap(color.rgb * exposure) / uncharted2Tonemap(vec3(whitePoint));
```

Comment:
I thought it made it easier to read, but sure I can add a comment and a link for reference. 

i.e : http://frictionalgames.blogspot.com/2012/09/tech-feature-hdr-lightning.html